### ISSUE TITLE
Bug with "relevance" label translations in Spanish language exhibit

### ISSUE THREAD

**Issue by caaster** [projectblacklight/spotlight#2559]
In the relevance pulldown on the Spanish language version of: https://exhibits.stanford.edu/garcia-sampedro_rolls, the word “relevance” is displayed — this is an English word, and the Spanish word “relevancia” should display instead. In addition, the pulldown label should be: Ordenar por relevancia.

See example: https://exhibits.stanford.edu/es/garcia-sampedro_rolls/browse/piano-rolls
![Screen Shot 2020-08-11 at 4 43 15 PM](https://user-images.githubusercontent.com/3210225/89959656-cd870280-dbf1-11ea-9774-e46d168fd5d3.png)

Additional observations from @ggeisler below:

Relevance issue appears to be a bug. The translation should go here: https://exhibits.stanford.edu/garcia-sampedro_rolls/translations/edit and then near the bottom of the Search field labels tab, there is a Sort Fields section.
But when I plug in the translation text, it appears it was saved but when you navigate to another page and go back to https://exhibits.stanford.edu/garcia-sampedro_rolls/translations/edit the translated text reverts to “Relevance”.

The pulldown label comes from the same translation. So both occurrences are due to the same bug

**Comment by cbeer** [comment]
I went to that page and successfully changed the label and it appears as expected both places.  I updated this field:

<img width="814" alt="Screen Shot 2020-08-11 at 16 52 49" src="https://user-images.githubusercontent.com/111218/89960173-18ede080-dbf3-11ea-8d12-24f1180b4e1d.png">

and (perhaps importantly?) hit the save button on the same tab.

**Comment by ggeisler** [comment]
@caaster I think I gave you incorrect info. After @cbeer tested and was successful, I tried again and couldn't recreate the problem. I think I just forgot to save after adding the translated string.

And the reason you thought there was a problem in the first place was that there was a translated value in the form for Relevance, but it was "Relevance" (the English word was in the translated input field). So someone just put that in by accident at some point, and when I saw that I thought there was a problem with the translated value being reset to the English default. And my forgetting to save when I tested just confirmed what I thought was the problem.

But Chris made the change when he was testing it so I believe all is good now.
